tests/queries/0_stateless/02357_file_default_value.sql

```diff
@@ -0,0 +1,3 @@
+SELECT file('nonexistent.txt'); -- { serverError 107 }
+SELECT file('nonexistent.txt', 'default');
+SELECT file('nonexistent.txt', NULL);
```

Comment:
Let's also add tests with files that exists and with non-constant first argument